lib/shared/languageSpecificTranslation.js

```diff
@@ -0,0 +1,57 @@
+import bcp47 from 'bcp47'
+import translations from '../../lib/language_specific_translation/translations.js'
+
+const translationMap = new Map(
+  Object.entries(translations.translation).map(([key, value]) => [
+    key,
+    new Map(Object.entries(value)),
+  ])
+)
+
+/**
+ * Checks if the document language is specified and not English
+ *
+ * @param {string | undefined} language - The language expression to check
+ * @returns {boolean} True if the language is valid, false otherwise
```

Comment:
We are missing the English part here. Not the best suggestion:
```suggestion
 * @returns {boolean} False if language is English, true if the language is valid, false otherwise
```